include/UnixStamp.hpp

```diff
@@ -23,19 +25,20 @@
 class UnixStamp
```

Comment:
Yep, and this magically equal to the number of days between 1 March and 1 August :)

Reply:
Yeah? What a coincidence.

It's 306 (in `(306 * mp + 5) / 10`) divided 2. And `5 / 2` turns into 2, because of integer division.

What's special about August 1st? What's the mathematical sense in March to August distance?

Reply:
The answer is simple - it is magic!